egress/egress-only-tracking.yaml

```diff
@@ -0,0 +1,74 @@
+components:
+  schemas:
+    EgressOnlyTacking:
+      description: |-
+        The list of structures containing Egress offset and it's length to be placed in the packet.
+      type: object
+      required: [port_name]
+      properties:
+        port_name:
+          description: Name of the Rx port where egress only tracking enabled.
```

Comment:
Name of the received port where this egress tracking rule/specification has to be applied.

Reply:
Updated.